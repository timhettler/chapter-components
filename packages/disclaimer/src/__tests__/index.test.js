import React from 'react';
import Block from '../';
import renderer from 'react-test-renderer';

it(`${Block.name} renders correctly`, () => {
  const tree = renderer.create(<Block copy="FOO" />).toJSON();
  expect(tree).toMatchSnapshot();
});
