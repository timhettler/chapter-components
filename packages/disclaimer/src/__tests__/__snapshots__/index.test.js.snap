// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Disclaimer renders correctly 1`] = `
<div
  className="disclaimer is-dark"
>
  <p>
    FOO
  </p>
</div>
`;
